packages/java/parser-jvm-plugin-backbone/src/main/java/dev/hilla/parser/plugins/backbone/BackbonePlugin.java

```diff
@@ -1,15 +1,23 @@
 package dev.hilla.parser.plugins.backbone;
 
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jdk8.Jdk8Module;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import com.fasterxml.jackson.module.paramnames.ParameterNamesModule;
+
 import dev.hilla.parser.core.AbstractCompositePlugin;
 import dev.hilla.parser.core.PluginConfiguration;
 
 public final class BackbonePlugin
         extends AbstractCompositePlugin<PluginConfiguration> {
-
     public BackbonePlugin() {
         super(new EndpointPlugin(), new EndpointExposedPlugin(),
                 new MethodPlugin(), new MethodParameterPlugin(),
-                new EntityPlugin(), new FieldPlugin(),
-                new TypeSignaturePlugin(), new JSONPlugin());
+                new EntityPlugin(),
+                new PropertyPlugin(new ObjectMapper()
+                        .registerModule(new ParameterNamesModule())
+                        .registerModule(new Jdk8Module())
+                        .registerModule(new JavaTimeModule())),
```

Comment:
I'd prefer use the same (configurable) `ObjectMapper` on both the parser and the controller

Reply:
Yes, I'm working on it but it's gonna be the next PR to not bloat this one.